### ISSUE TITLE
The Prerequisites section of each test contains a ...

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-harmonised-sds#3]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The Prerequisites section of each test contains a verbose requirement validity against xml schema.
Since the xml schema is now the standard ISO 19139, it should simply reference the ats-metadata requirement.

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
